### ISSUE TITLE
Redis driver is ko

### ISSUE THREAD

**Issue by Marc-pi** [pi-engine/pi#1418]
at fresh install, cannot go up to DB setting >>> blank page

**Comment by Marc-pi** [comment]
yes, confirmed. i don't know if it is a (shared) server issue or Pi issue

**Comment by voltan** [comment]
I think its pi issues

**Comment by voltan** [comment]
On my last test :it work by this steps : 

1. install redis on fedora linux 
2. start and enable redis service
3. install php redis client on php core
4. set pi config to use redis

**Comment by voltan** [comment]
Here I am on test, by install last version redis on server and config Pi ( on both of engine.php and service.cache.php files ) to use redis. its work fine

![Screenshot_20190611_103735](https://user-images.githubusercontent.com/562083/59247564-09f95000-8c35-11e9-82c4-5836fe9dcfa1.png)
